### ISSUE TITLE
Numérotation automatique des chapitres et sections

### ISSUE THREAD

**Issue by jppellet** [edunumsec2/book#74]
### Page

Toutes

### Quel est le problème relevé ?

Les sections et sous-sections en Markdown doivent être numérotées à la main.

Ça pose problème d'une part parce qu'on est des humains et qu'on se plante (je viens de corriger un 3.3 qui suivait directement un 3.1) et d'autre part parce qu'une insertion de sections ou sous-sections demande une renumérotation manuelle fastidieuse de tout ce qui suit…

### Quelle est votre suggestion ?

Faire en sorte que la numération soit automatique

### Code of Conduct

- [X] J'accepte de suivre le code de conduite.

**Comment by elliotvaucher** [comment]
Hello @jppellet. Merci pour l'issue. Je me mets là-dessus en fin de semaine, après les formations. C'est juste une petite gymnastique à faire avec les index.md. Il faut ajouter l'option `:numbered:`, puis retirer celles qui ont été faites à la main dans les chapitres. 

À plus, 

Elliot.

**Comment by jppellet** [comment]
Oui, mais il faut trouver comment ne pas numéroter l’intro et le conclusion de chaque thématique, aussi. Ou bien commencer la numérotation à 0… Ce serait bien que ce soit uniforme dans tout modulo.
